### ISSUE TITLE
No module named 'six'

### ISSUE THREAD

**Issue by marcsit** [AndrewAnnex/SpiceyPy#197]
Andrew, 

while pipping SpiceyPy from a fresh OS with a fresh Python 3 I got the following error:

```
  Downloading spiceypy-2.0.0.tar.gz (221kB)
    100% |████████████████████████████████| 225kB 367kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/sc/45xrb78x3372prlpzxr0wk2m0000gn/T/pip-build-n8nomi05/spiceypy/setup.py", line 29, in <module>
        import getspice
      File "/private/var/folders/sc/45xrb78x3372prlpzxr0wk2m0000gn/T/pip-build-n8nomi05/spiceypy/getspice.py", line 58, in <module>
        import six.moves.urllib as urllib
    ModuleNotFoundError: No module named 'six'
```
Needless to say it was solved by pip installing six, but just for you to know if you want to include it as a dependency.

Cheers,

**Comment by marcsit** [comment]
Same thing happens with numpy when I import spiceypy, solved from my side of course.

**Comment by AndrewAnnex** [comment]
yeah there is something not exactly right with my setup.py file, on some systems I am able to do a fresh install with no issues. I will put this on the todo list

**Comment by AndrewAnnex** [comment]
So I have a fix in #198, it looks like I needed to update the `setup_requires` keyword in my setup.py

Currently I do have that branch installing numpy, however this is a thing other libraries like scikit-learn explicitly do not do as a pip install of numpy could be less performant than more careful installs or what is distributed in anaconda (ie. things like MKL). I will have to think about this for a bit.

**Comment by marcsit** [comment]
Thanks Andrew! You're awesome ;)

**Comment by AndrewAnnex** [comment]
merged #198 so this issue is closed for now!
